sunpy/coordinates/tests/test_transformations.py

```diff
@@ -5,10 +5,50 @@
 from astropy.coordinates import SkyCoord, get_body_barycentric
 from astropy.time import Time
 
-from sunpy.coordinates import Helioprojective, HeliographicStonyhurst, HeliographicCarrington, get_sun_L0
+import sunpy.coordinates.transformations as trans
+from sunpy.coordinates import (Helioprojective, HeliographicStonyhurst,
+                               HeliographicCarrington, get_sun_L0,
+                               Heliocentric)
 from sunpy.time import parse_time
 
 
+def test_new_hcc_to_hgs():
+    # Generate 10 random coordinates
+    for i in range(50):
+        x = (np.random.rand() - 0.5) * u.km
+        y = (np.random.rand() - 0.5) * u.km
+        z = (np.random.rand() - 0.5) * u.km
```

Comment:
I may have not explain myself well before, what I meant is that you are only testing coordinate transformations with a variation of +/- 0.5 km, where in the sun that's is almost imperceptible. I wonder whether even we have such precision to distinguish things in that range.
Maybe:
```python
x = (np.random.rand() - 0.5) * np.random.randint(100) * u.Mm
```
will give you values between +/- 500 Mm.

Reply:
This transformation is just a rotation though, so the absolute scaling of the vectors shouldn't matter. I can swap to Mm though.